src/Components/test/E2ETest/ServerExecutionTests/ServerAuthTest.cs

```diff
@@ -55,7 +55,7 @@ void AssertState(string username)
 
         private void PerformReconnection()
         {
-            ((IJavaScriptExecutor)Browser).ExecuteScript("Blazor._internal.forceCloseConnection()");
+            ((IJavaScriptExecutor)Browser).ExecuteScript($"fetch('/WebSockets/Interrupt?WebSockets.Identifier={SessionIdentifier}')");
```

Comment:
LOL dope.

Reply:
Should we be concerned about the result of this? If fetch failed (lets say a 404) how would the tests fail? I'd really like to make sure thing fail in a reasonable way.

Reply:
We can change it to:
```csharp
((IJavaScriptExecutor)Browser).ExecuteScript($"fetch('/WebSockets/Interrupt?WebSockets.Identifier={SessionIdentifier}').then(response => window.disconnection = response.ok)");
```

And wait until we see it populated

Reply:
Done, added a couple of utilities for making JS usage easier too as part of fixing this.